node/src/reactor/main_reactor/catch_up.rs

```diff
@@ -28,6 +28,49 @@ pub(super) enum CatchUpInstruction {
     CommitUpgrade,
 }
 
+/// This diagram represents how the components involved in the sync process interact with each
```

Comment:
Should this chart maybe be applied to the definition of the `MainReactor` in `main_reactor.rs:84`?

Reply:
I'm reluctant to attach it directly to `MainReactor` as it is responsible for many more things (diagnostics port, even stream, metrics...). The chart is focused on the interactions between components used in the so called "fast-sync" process.

However, I agree that the current location isn't perfect, so maybe I can move it to `MainReactor` and just add an appropriate comment about what is shown in the graph, wdyt?

Reply:
Updated in 8ab8c97a34adda6010b29cb828ad51e9bb6832c6.

This is how it now looks like in the docs:

<kbd><img src="https://user-images.githubusercontent.com/88321181/212866917-297a8c06-7cfa-4059-8d56-c99ea2e66d06.png" /> </kbd>